testsuite/integration-tests/pom.xml

```diff
@@ -274,6 +274,12 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.hibernate.validator</groupId>
+            <artifactId>hibernate-validator</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
What's the link between tests failing with security manager enabled and the need for explicit hibernate-validator dependency here? Is it to control the exact version being used? And if that's the case, is it really a problem to have code that runs regardless the version of the hibernate-validator that comes from the selected target container?